sentry-log/src/lib.rs

```diff
@@ -41,6 +41,7 @@
 #![doc(html_favicon_url = "https://sentry-brand.storage.googleapis.com/favicon.ico")]
 #![doc(html_logo_url = "https://sentry-brand.storage.googleapis.com/sentry-glyph-black.png")]
 #![warn(missing_docs)]
+#![allow(clippy::match_like_matches_macro)]
```

Comment:
Instead of allowing this, can we just fix this?

Reply:
not unless we break the MSRV, since `matches!` is a 1.42 feature.